csp-tech-radar.csv

```diff
@@ -23,6 +23,7 @@ Kafka,adopt,platforms,TRUE,"Apache Kafka continues to build momentum as an open
 Windows Containers,assess,platforms,TRUE,"To achieve legacy in a box for many of our CSP applications that target the .Net Framework, Windows Containers should be explored as a possible solution."
 Oracle,hold,platforms,TRUE,Proprietary software is discouraged by the Open Source Software culture.
 Microsoft Proprietary Platforms,hold,platforms,TRUE,Proprietary software is discouraged by the Open Source Software culture. Basically anything that requires Windows Server operating system to run.
+MySQL,adopt,platforms,TRUE,"MySQL is a fast, reliable and stable open source database management system. It has most of the features which oracle provides. This is being used by PD/PMP."
```

Comment:
It think it would be good to highlight the fact that it is relational database and is suitable for use cases where persisting data in a relational way is beneficial.

Reply:
I was surprised to see this as a suggestion for our radar since it's already so widely used, but I guess it's a question of perspective. It makes sense in the context of moving away from Oracle as described!